KarhooSDK.xcodeproj/project.pbxproj

```diff
@@ -4314,7 +4326,7 @@
 				CODE_SIGN_IDENTITY = "";
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "";
 				CODE_SIGN_STYLE = Automatic;
-				CURRENT_PROJECT_VERSION = 28;
+				CURRENT_PROJECT_VERSION = 29;
```

Comment:
Did you bump the version in preparation for the release? If so, we'll need to remember not to bump again in 2 week's time. I'd rather bump the version then, or at least let the team know this has been changed already

Reply:
In last commit I reverted those changes, so we keep release process untouched.